### PR TITLE
TF-725 Add inline image to composer web

### PR DESCRIPTION
**Pull request by dab246**
### Resolve
- Use `cid` with `blod_id`
- Only error showing on `Gmail`

Screen Shot 2022-07-20 at 11 16 27

### Result Demo

https://user-images.githubusercontent.com/80730648/179956799-8d214592-8a43-45f2-939a-273a46ec8c93.mp4

### CODE REVIEW

**Comment by hoangdat** [comment]
@dab246 what any issue in the eml? Why `gmail` and `apple mail` can not show our inline image :)

**Comment by hoangdat** [comment]
after discussion with designer, please remove the image picker at the moment, we will support it later @dab246

File: lib/features/composer/domain/usecases/download_image_as_base64_interactor.dart [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+import 'package:dartz/dartz.dart';
+import 'package:tmail_ui_user/features/composer/domain/repository/composer_repository.dart';
+import 'package:tmail_ui_user/features/composer/domain/state/download_image_as_base64_state.dart';
+
+class DownloadImageAsBase64Interactor {
+  final ComposerRepository _composerRepository;
+
+  DownloadImageAsBase64Interactor(this._composerRepository);
+
+  Future<Either<Failure, Success>> execute(String url) async {
```
Comment (hoangdat) [inline lib/features/composer/domain/usecases/download_image_as_base64_interactor.dart:11]: I think you should you Stream, first item is Loading state. Maybe we should showing loading in the position of inline image when downloading? WDYT?

Comment (dab246) [reply lib/features/composer/domain/usecases/download_image_as_base64_interactor.dart:11]: Ok. I think that's quite reasonable.

**Review by hoangdat — APPROVED** [review]

**Pull request merged**